src/applications/vaos/services/appointment/index.js

```diff
@@ -199,9 +199,12 @@ export function getVARClinicId(appointment) {
  * @returns {String} The location id where the video appointment is located
  */
 export function getVideoAppointmentLocation(appointment) {
+  const serviceResource = appointment.contained.find(
+    res => res.resourceType === 'HealthcareService',
+  );
```

Comment:
Should we consider creating a helper function for this in the appointment service?  Maybe something called `getHealthcareService()` if we foresee needing to use it frequently.

Reply:
I looked for other places where we were doing something similar and didn't find any, so I ended up not making a helper. But I agree that's something to consider if we end up needing this more often.